datarepo-clienttests/src/main/java/runner/TestRunner.java

```diff
@@ -199,6 +215,10 @@ void executeTestConfigurationNoGuaranteedCleanup() throws Exception {
             testScriptSpecification.description);
       }
     }
+    if (disruptionThreadPool != null) {
+      logger.debug("Force shut down of the disruption thread pool");
+      disruptionThreadPool.shutdownNow();
+    }
```

Comment:
Could you add an awaitTermination/logger.error call for the disruption thread pool, similar to the user journey thread pools? That way we'll at least get an error logged if there are problems shutting down the pool.